contrib/completions/zsh/kubectl

```diff
@@ -0,0 +1,214 @@
+#!/bin/zsh
```

Comment:
Yes, it's just a collection of functions that zsh's bash-completion emulation does not provide or which are incomplete. Will create an independent PR.